particles/Chooser/Chooser.js

```diff
@@ -23,6 +23,7 @@ defineParticle(({particle: {Particle, ViewChanges, StateChanges, SlotChanges}})
         var inputList = await views.get('choices').toList();
         var outputList = await views.get('resultList').toList();
         var result = [...difference(inputList, outputList)];
+        if (result.length > 0) this.relevance = 10;
```

Comment:
I'm in the habit of expanding all conditionals into two-lines with braces:
`if (...) {
}`
I'm not sure what Google's official readability guidelines say.